code/index.html

```diff
@@ -5,26 +5,106 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
     <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+    
+    <div class="hero-image" id="hero-image">
+      <div class="hero-text">
+        <h1>Baking Classes</h1>
+        <p>At School of Baking</p>
+      </div>
+    </div>
+
+  <section class="middle-text">
+    <div class="muffin1">
+    <img src="muffin.png" alt="muffin"></div>
+    <div class="muffin2">
+      <img src="muffin.png" alt="muffin"></div>
+    <h3>Welcome to School of Baking!</h3>
+    <p>Have you always dreamt of becoming a baker? Or perhaps thought of picking baking up as a hobby? </p>
+    <p>No matter your personal goals, School of Baking offers several different courses in various areas of baking.</p>
+      <p>Learn everything from making the perfect loaf
+      to decorating a wedding cake. </p>
+    <p><b>We adjust to your schedule.</b> Come and learn at different times, all according to what suits you.</p> 
+    <p>See our <b>FAQ</b> for more information, or sign up and get a look at what we've got to offer!</p>
+  </section>
+
+ 
+ 
+    <form class="modal">
+      <div class="container">
+        <h1>Sign Up</h1>
+        <p>Please fill in this form to create an account.</p>
+        <hr>
+        <label for="email"><b>Email</b></label>
+        <input type="text" placeholder="Enter Email" name="email" required>
```

Comment:
One idea could be to use the type email here instead of text. This way you could add a validation to ensure that the email address is correctly entered.  Learn more on https://www.w3schools.com/tags/att_input_type_email.asp